form/create_custom_field_type.rst

```diff
@@ -131,6 +65,14 @@ implement the ``getParent()`` method (Symfony will make the type extend from the
 generic :class:`Symfony\\Component\\Form\\Extension\\Core\\Type\\FormType`,
 which is the parent of all the other types).
 
+.. note::
+
+    The PHP class extension mechanism and the Symfony form field extension
```

Comment:
> The PHP class extension mechanism

I don't know what this is. Do you mean inheritance?

Reply:
I agree we should change that, but I propose to do it in https://github.com/symfony/symfony-docs/pull/13488 which targets 3.4 instead. You're review is welcome there too :)